audformat/core/table.py

```diff
@@ -479,7 +478,7 @@ def _load_csv(self, path: str):
             if column.scheme_id is not None:
                 dtypes[column_id] = schemes[column.scheme_id].to_pandas_dtype()
             else:
-                dtypes[column_id] = 'str'
+                dtypes[column_id] = object
```

Comment:
Not related to this PR, but I think it's safer to use `object` here.

Reply:
For now it doesn't matter, but if `pandas` changes the behavior for `str` at some point, yes.